appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 1.0.{build}
```

Comment:
Current version of `Hangfire.Autofac` package is `2.2.0`. Could you change version field to `2.3.0-build-0{build}`